sort/challenges-12.test.js

```diff
@@ -0,0 +1,372 @@
+'use strict';
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 1
+//
+// Write a function named sortBackwards that takes in an array of numbers and returns the same array,
+// with the numbers sorted, highest to smallest.
+// ------------------------------------------------------------------------------------------------
+
+const sortBackwards = (nums) => {
+  return nums.sort((a, b) => {
+    if (a > b) {
+      return -1;
+    } else if (a < b) {
+      return 1;
+    } else {
+      return 0;
+    }
+  });
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 2
+//
+// Write a function named alphabetize that takes in an array of strings and returns the same array,
+// with the strings sorted alphabetically.
+//
+// In this alphabetization, capital letters come before lower case letters.
+// For example, ['Alphabet', 'Zebra', 'alphabet', 'carrot'] is correctly sorted.
+// ------------------------------------------------------------------------------------------------
+
+const alphabetize = (strings) => {
+  return strings.sort();
+};
+
+// ------------------------------------------------------------------------------------------------
+// CHALLENGE 3
+//
+// Write a function named sortByLength that takes in an array of strings and returns the same array,
+// with the strings sorted by their length, lowest to highest.
+// ------------------------------------------------------------------------------------------------
+
+const sortByLength = (strings) => {
+  return strings.sort((a, b) => {
+    if (a.length > b.length) {
+      return 1;
+    } else if (a.length < b.length) {
+      return -1;
+    } else {
+      return 0;
+    }
+  });
+};
+
```

Comment:
one line, similar to challenge 1, `return strings.sort( (a,b) => a.length - b.length );`